### DISCUSSION TITLE
How to locate to the precise position of error ?

### DISCUSSION THREAD

**Discussion by 7tosmoke** [PKU-Alignment/omnisafe#290] [Q&A]
I'm using my own custom_env. During coding, I meet many error but only can be shown at the top funtion. The following pic is an example.
![1700647063693](https://github.com/PKU-Alignment/omnisafe/assets/53805503/f222856f-ad99-4508-b2c5-26c6ad528801)
I wonder if I can get the precise position of error.
Thanks a lot!

**Comment by Gaiejj** [comment]
I don't really have a way to determine the location of the error based only on the information you provided. Can you provide more detailed information such as your customized environment code, or other possible error messages so that we can reproduce your error?
